test/e2e/case18_compliance_api_test.go

```diff
@@ -1728,12 +1746,15 @@ var _ = Describe("Test the compliance events API", Label("compliance-events-api"
 		)
 
 		Describe("Test the /api/v1/reports/compliance-events endpoint", func() {
+			csvEndpoints := "http://localhost:8385/api/v1/reports/compliance-events"
 			It("should send CSV file in http response", func(ctx context.Context) {
-				endpoints := "http://localhost:8385/api/v1/reports/compliance-events"
-
-				req, err := http.NewRequestWithContext(ctx, http.MethodGet, endpoints, nil)
+				req, err := http.NewRequestWithContext(ctx, http.MethodGet, csvEndpoints, nil)
 				Expect(err).ShouldNot(HaveOccurred())
 
+				// Set auth token
+				token := getToken(ctx)
```

Comment:
Could you cache this at the top of the describe so it can be reused for all the tests? It's expensive to keep calling the API.